worlds/baylands.sdf

```diff
@@ -73,22 +74,30 @@
     <scene>
       <ambient>0.8 0.5 1</ambient>
       <grid>false</grid>
+      <sky>
+        <clouds>true</clouds>
+      </sky>
+      <shadows>1</shadows>
     </scene>
-    <light type="directional" name="sun">
-      <cast_shadows>true</cast_shadows>
-      <pose>0 0 100 0 0.75 0</pose>
-      <diffuse>0.8 0.8 0.8 1</diffuse>
-      <specular>0.0 0.0 0.0 1</specular>
-      <intensity>5</intensity>
+    <light name='sunUTC' type='directional'>
+      <pose>0 0 500 0 -0 0</pose>
+      <cast_shadows>false</cast_shadows>
+      <intensity>1</intensity>
+      <direction>0.001 0.625 -0.78</direction>
+      <diffuse>0.904 0.904 0.904 1</diffuse>
+      <specular>0.271 0.271 0.271 1</specular>
       <attenuation>
-        <range>100</range>
-        <constant>0.9</constant>
-        <linear>0.01</linear>
-        <quadratic>0.001</quadratic>
+        <range>2000</range>
+        <linear>0</linear>
+        <constant>1</constant>
+        <quadratic>0</quadratic>
       </attenuation>
-      <direction>-0.5 0.1 -0.9</direction>
+      <spot>
+        <inner_angle>0</inner_angle>
+        <outer_angle>0</outer_angle>
+        <falloff>0</falloff>
+      </spot>
```

Comment:
You are using a directional light, spot is not needed here and can be confusing as to what light source you are trying to use. Please follow the SDF guide here for associated used params for a directional light: http://sdformat.org/spec?ver=1.11&elem=light